test/h5test.c

```diff
@@ -771,16 +629,238 @@ h5_fileaccess_flags(unsigned flags)
     return H5I_INVALID_HID;
 } /* end h5_fileaccess_flags() */
 
-/*-------------------------------------------------------------------------
- * Function:    h5_get_libver_fapl
- *
- * Purpose:     Sets the library version bounds for a FAPL according to the
- *              value in the constant or environment variable "HDF5_LIBVER_BOUNDS".
- *
- * Return:      Success:    0
- *              Failure:    -1
- *
- *-------------------------------------------------------------------------
+/*
+ * Modifies the File Driver set on the given File
+ * Access Property List according to the HDF5_DRIVER
+ * or HDF5_TEST_DRIVER environment variables
+ */
+herr_t
+h5_get_vfd_fapl(hid_t fapl)
```

Comment:
The VFD testing was messed up around the 1.14.2 release when this function was removed. It was restored in a later release and is worth bringing it back into this branch for VFD testing in the future.